actionpack/test/dispatch/test_request_test.rb

```diff
@@ -54,9 +54,7 @@ class TestRequestTest < ActiveSupport::TestCase
   end
 
   test "does not complain when Rails.application is nil" do
-    Rails.stubs(:application).returns(nil)
```

Comment:
Yeah lets change the message here (and in the other test you're referring to - if that's appropriate).